src/app.jsx

```diff
@@ -30,25 +32,27 @@ const App = () => {
   }, []);
 
   return (
-    <ErrorBoundary>
-      <Router historyRef={historyRef.current}>
-        <section className={styles.app_wrapper}>
-          <GordonHeader onDrawerToggle={onDrawerToggle} />
-          <GordonNav onDrawerToggle={onDrawerToggle} drawerOpen={drawerOpen} />
-          <main className={styles.app_main}>
-            <>
-              <BirthdayMessage />
-              <AppRedirect />
-              <Routes>
-                {routes.map((route) => (
-                  <Route key={route.path} path={route.path} element={route.element} />
-                ))}
-              </Routes>
-            </>
-          </main>
-        </section>
-      </Router>
-    </ErrorBoundary>
+    <CssVarsProvider theme={theme}>
```

Comment:
Since you started working on this, all the `Provider`-type components were relocated to `src/components/Providers/index.tsx`. Could you please rebase (or merge) with the latest `develop` and move this to that file?